### PR TITLE
Add EventListener to Driver

### PR DESCRIPTION
**Pull request by KazuCocoa**
close Add EventListener support #339 #366

- fix reviewed point in #366
- add tests to check `listener`'s set value

### CODE REVIEW

**Comment by KazuCocoa** [comment]
@jlipps Who could review this PR ? and merge this ?
(If any review and merge processes are defined for current this library, I'd like to know it.)
cc: @bootstraponline

**Comment by bootstraponline** [comment]
I think I granted you merge rights. I moved on a long time ago so I'll defer to jlipps on how to handle the release process.

**Comment by jlipps** [comment]
@KazuCocoa the problem is you're probably in a better position to review this than I am, since I never really worked on this library :-) If you don't mind making it better, by all means welcome to the project and please merge/release as necessary assuming you believe everything to be well-tested enough.

**Comment by KazuCocoa** [comment]
Thanks for quick responses! @bootstraponline @jlipps 
I understood current status. I'll commit this project as much I can.

File: ios_tests/appium.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [caps]
 platformName = "ios"
-platformVersion = "9.2"
+platformVersion = "8.3"
```
Comment (KazuCocoa) [inline ios_tests/appium.txt]: Update `platformVersion`s and related documentations in https://github.com/appium/ruby_lib/pull/389/commits/f93252ebdb4fc699139d75ac86378c8002ff2665

**Comment by KazuCocoa** [comment]
tests associated this PR is passed => https://travis-ci.org/KazuCocoa/ruby_lib/builds/174785405

**Comment by KazuCocoa** [comment]
I'll merge this and close related issues.

**Pull request merged**